system/languages/en.php

```diff
@@ -184,6 +184,8 @@
 		"edit-day" => "edit day",
 		"invalid-money-amount" => "Not a valid amount of money",
 		"required-field-missing" => "Required field missing",
+		"email-field-invalid" => "Invalid email address supplied",
+		"html-field-contains-script" => "HTML cannot contain <script> tag. Your computer or browser might be infected by a virus",
```

Comment:
I find this message slightly confusing..
If I as a user would copy paste some html I wanted to use and I get a notification that my computer might be infected by a virus. I would get worried, perhaps for the wrong reasons. 
Something along these lines perhaps? `"HTML <script> tags are not allowed. They are considered unsafe as they can be used for hacks or virusses."`

Reply:
Ah, just read this comment in the code above, this explains the reasoning behind the stringent message. But I think the message could still be made more clear.

```
 (and these have been seen to be
 			// automatically inserted into requests by
 			// viruses...).
```

Reply:
Yeah, I didn't spend too much effort on the message, since it should not normally be shown. Do you have a specific suggestion for a better message?

Reply:
Something like this? 
"HTML <script> tags are not allowed. They are considered unsafe as they can be used for hacks or viruses. If you not add them yourself your computer could even be infected."

Reply:
Thanks, I've used your suggestion.